ts/Extensions/PriceIndication.ts

```diff
@@ -74,17 +74,104 @@ declare global {
 
 /**
  * @declare   Highcharts.SeriesLastVisiblePriceLabelOptionsObject
+ * @extends   yAxis.crosshair.label
+ * @since     7.0.0
  * @apioption plotOptions.series.lastVisiblePrice.label
  */
 
 /**
- * Enable or disable the label.
+ * @since     7.0.0
+ * @apioption plotOptions.series.lastVisiblePrice.label.align
+ */
+
+/**
+ * @since     7.0.0
+ * @apioption plotOptions.series.lastVisiblePrice.label.backgroundColor
+ */
+
+/**
+ * The border color for the lastVisiblePrice label.
  *
- * @type      {boolean}
+ * @type      {Highcharts.ColorType}
+ * @since     7.0.0
+ * @product   highstock
+ * @apioption plotOptions.series.lastVisiblePrice.label.borderColor
+ */
+
+/**
+ * The border corner radius of the lastVisiblePrice label.
+ *
+ * @type      {number}
+ * @default   3
+ * @since     7.0.0
  * @product   highstock
+ * @apioption plotOptions.series.lastVisiblePrice.label.borderRadius
+*/
+
+/**
+ * Flag to enable lastVisiblePrice label.
+ *
+ *
+ * @type      {boolean}
  * @default   false
+ * @since     7.0
+ * @product   highstock
  * @apioption plotOptions.series.lastVisiblePrice.label.enabled
+ */
+
+/**
+ * A format string for the lastVisiblePrice label. Defaults to `{value}` for
+ * numeric axes and `{value:%b %d, %Y}` for datetime axes.
  *
+ * @type      {string}
+ * @since     7.0
+ * @product   highstock
+ * @apioption plotOptions.series.lastVisiblePrice.label.format
+*/
+
+/**
+ * @since     7.0.0
+ * @apioption plotOptions.series.lastVisiblePrice.label.formatter
+ */
+
+/**
+ * @since     7.0.0
+ * @apioption plotOptions.series.lastVisiblePrice.label.padding
+ */
+
+/**
+ * @since     7.0.0
+ * @apioption plotOptions.series.lastVisiblePrice.label.shape
+ */
+
+/**
+ * Text styles for the lastVisiblePrice label.
+ *
+ * @type      {Highcharts.CSSObject}
+ * @default   {"color": "white", "fontWeight": "normal", "fontSize": "11px", "textAlign": "center"}
+ * @since     7.0
+ * @product   highstock
+ * @apioption plotOptions.series.lastVisiblePrice.label.style
+ */
+
+/**
+ * The border width for the lastVisiblePrice label.
+ *
+ * @type      {number}
+ * @default   0
+ * @since     7.0
+ * @product   highstock
+ * @apioption plotOptions.series.lastVisiblePrice.label.borderWidth
+*/
+
+/**
+ * Padding inside the lastVisiblePrice label.
+ *
+ * @type      {number}
+ * @default   8
+ * @since     7.0
+ * @product   highstock
+ * @apioption plotOptions.series.lastVisiblePrice.label.padding.
```

Comment:
Change:
> `plotOptions.series.lastVisiblePrice.label.padding.`

To:
> `plotOptions.series.lastVisiblePrice.label.padding`

Otherwise we get:

![Zrzut ekranu 2021-04-8 o 18 02 35](https://user-images.githubusercontent.com/1453926/114059407-a07c1400-9894-11eb-9761-9f890648b89f.png)

Reply:
Clue: it is possible to generate local API using `npx gulp jsdoc-watch`. I know it takes time to generate, but it is worth checking at least once before committing.